firmware/arduino/chessboard/chessboard.ino

```diff
@@ -36,6 +36,11 @@ int yMotor[4] = {MOTOR2_STEP_PIN, MOTOR2_DIR, MOTOR2_MS1, MOTOR2_MS2};
 #define Y_AXIS_ENDSTOP_SWITCH 32
 #define CHESS_TIMER_BUTTON 4
 
+// Distance Definitions
+#define MILLIMETERS_PER_SQUARE 200
+#define STEPS_PER_MILLIMETER 200
+float stepsPerSpace = MILLIMETERS_PER_SQUARE * STEPS_PER_MILLIMETER;
```

Comment:
I'll defer to @LSaber36 's opinion on this, but 1) stepsPerSpace should probably be constant since it doesn't change during the program, and 2) since it's a global constant maybe it should be named STEPS_PER_SPACE

Reply:
It will be a constant, but I suggested this so we would have less calculations to do outside of the program. If we need to fine tune that, which I'm pretty sure we will, we'd need to calculate that outside of the program. That's why I suggested doing it this way. The cost is negligible since it's only one float and calculations only show up during setup()

Reply:
I just meant making it `const float STEPS_PER_SPACE` instead of what it is now! So it's clear in the program that it is not a variable